src/components/simpleGrid/index.tsx

```diff
@@ -4,20 +4,15 @@ import {
   SimpleGridProps,
 } from '@chakra-ui/react';
 
-type Props = Pick<
+export type Props = Pick<
```

Comment:
This is globally re-exported by the package and needs to be prefixed by the component name:
```suggestion
export type SimpleGridPropsProps = Pick<
```

Reply:
@marko-markovic-smg this was not addressed before merging the PR. Please address this feedback.